udata/harvest/forms.py

```diff
@@ -39,6 +39,9 @@ def pre_validate(self, form):
                     msg = 'Unknown filter key "{0}" for "{1}" backend'
                     msg = msg.format(f['key'], backend.name)
                     raise validators.ValidationError(msg)
+
+                f['value'] = f['value'].encode('utf-8') #Fix encoding error
```

Comment:
You should check that the expected type is a string and make use of [`udata.utils.safe_unicode`](https://github.com/opendatateam/udata/blob/master/udata/utils.py#L277-L283):
```suggestion
            if isinstance(f['value'], basestring):  # Not tested, this test needs to be improved
                f['value'] = safe_unicode(f['value'])  # Fix encoding error
```

Reply:
"basestring", is the base for str and unicode, so should be fine!
I've opened this PR because we faced encoding issues when we used filters like this ( with special characters like í, ç, ã, etc.. ):
![image](https://user-images.githubusercontent.com/9539326/47316822-ecdd2600-d63f-11e8-8532-a1e4b4e9af45.png)

Tested with your solution, it's working as expected!